### ISSUE TITLE
Autocompletion and Formatting not working 

### ISSUE THREAD

**Issue by HenriBeck** [Dart-Code/Dart-Code#2370]
It seems that my auto-completion and formatting inside VSCode has stopped working.

Whenever I try to save a file, the auto-formatter popup shows up after a few seconds saying it's `Running Dart Formatter`, but it never finishes.
Additionally, the auto-completion and the hover information doesn't work anymore.

When I look at the dart process, it's at a 100% CPU power and seems to be stuck.

I have already tried reinstalling my flutter installation, but the error still persists.

Dart Analyzer Logs:
```
!! PLEASE REVIEW THIS LOG FOR SENSITIVE INFORMATION BEFORE SHARING !!

Dart Code extension: 3.9.1
Flutter extension: 3.9.1 (not activated)
VS Code: 1.44.2
Platform: mac
Workspace type: Flutter
Multi-root?: false
Dart SDK:
    Loc: /Users/henri.beck/flutter/bin/cache/dart-sdk
    Ver: 2.7.0
Flutter SDK:
    Loc: /Users/henri.beck/flutter
    Ver: 1.12.13+hotfix.8
HTTP_PROXY: undefined
NO_PROXY: undefined

Mon Apr 20 2020 [21:01:59 GMT+0200 (Central European Summer Time)] Log file started
[9:01:59 PM] [Analyzer] [Info] Spawning /Users/henri.beck/flutter/bin/cache/dart-sdk/bin/dart with args ["/Users/henri.beck/flutter/bin/cache/dart-sdk/bin/snapshots/analysis_server.dart.snapshot","--client-id=VS-Code","--client-version=3.9.1","--instrumentation-log-file=/Users/henri.beck/.dart-analyzerInstrumentationLogFile"]
[9:01:59 PM] [Analyzer] [Info] ..  with {"toolEnv":{"FLUTTER_HOST":"VSCode","PUB_ENVIRONMENT":"vscode.dart-code"}}
[9:01:59 PM] [Analyzer] [Info]     PID: 2351
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"1","method":"server.setSubscriptions","params":{"subscriptions":["STATUS"]},"clientRequestTime":1587409319272}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"2","method":"analysis.setPriorityFiles","params":{"files":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml"]},"clientRequestTime":1587409319273}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"3","method":"analysis.setSubscriptions","params":{"subscriptions":{"FOLDING":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml"],"OCCURRENCES":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml"],"OUTLINE":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml"]}},"clientRequestTime":1587409319274}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"4","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml":{"content":"name: sign_in_with_apple\ndescription: Flutter bridge to initiate Sign in with Apple (currently iOS and macOS only). Includes support for keychain entries as well as signing in with an Apple ID.\nversion: 2.0.0\nhomepage: https://github.com/aboutyou/dart_packages/tree/master/packages/sign_in_with_apple\nrepository: https://github.com/aboutyou/dart_packages\n\nenvironment:\n  sdk: \">=2.3.0 <3.0.0\"\n  flutter: \">=1.12.13+hotfix.8 <2.0.0\"\n\ndependencies:\n  meta: ^1.0.0\n\n  flutter:\n    sdk: flutter\n\ndev_dependencies:\n  flutter_test:\n    sdk: flutter\n\nflutter:\n  plugin:\n    platforms:\n      android:\n        package: com.aboutyou.dart_packages.sign_in_with_apple\n        pluginClass: SignInWithApplePlugin\n      ios:\n        pluginClass: SignInWithApplePlugin\n      macos:\n        pluginClass: SignInWithApplePlugin\n","type":"add"}}},"clientRequestTime":1587409319283}
[9:01:59 PM] [Analyzer] [Info] <== {"event":"server.connected","params":{"version":"1.27.4","pid":2432}}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"5","method":"analysis.setAnalysisRoots","params":{"excluded":[],"included":["/Users/henri.beck/Projects/AboutYou/dart_packages"]},"clientRequestTime":1587409319796}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"6","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml":{"edits":[{"offset":0,"length":0,"replacement":"","id":""}],"type":"change"}}},"clientRequestTime":1587409319797}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"7","method":"completion.setSubscriptions","params":{"subscriptions":["AVAILABLE_SUGGESTION_SETS"]},"clientRequestTime":1587409319798}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"8","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml":{"edits":[{"offset":0,"length":0,"replacement":"","id":""}],"type":"change"}}},"clientRequestTime":1587409319799}
[9:01:59 PM] [Analyzer] [Info] <== {"id":"1"}
[9:01:59 PM] [Analyzer] [Info] <== {"id":"2"}
[9:01:59 PM] [Analyzer] [Info] <== {"id":"3"}
[9:01:59 PM] [Analyzer] [Info] ==> {"id":"9","method":"flutter.setSubscriptions","params":{"subscriptions":{"OUTLINE":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml"]}},"clientRequestTime":1587409319874}
[9:01:59 PM] [Analyzer] [Info] <== {"id":"4","result":{}}
[9:02:00 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/profile/AndroidManifest.xml","errors":[]}}
[9:02:00 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/main/AndroidManifest.xml","errors":[]}}
[9:02:00 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/debug/AndroidManifest.xml","errors":[]}}
[9:02:00 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/android/src/main/AndroidManifest.xml","errors":[]}}
[9:02:00 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/pending_operations/analysis_options.yaml","errors":[]}}
[9:02:00 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/pending_operations/pubspec.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/state_queue/analysis_options.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/state_queue/pubspec.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/with_bloc/analysis_options.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/with_bloc/pubspec.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/with_bloc/analysis_options.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/with_bloc/example/pubspec.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/analysis_options.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/profile/AndroidManifest.xml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/main/AndroidManifest.xml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/debug/AndroidManifest.xml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/android/src/main/AndroidManifest.xml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/analysis_options.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/pubspec.yaml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/profile/AndroidManifest.xml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/main/AndroidManifest.xml","errors":[]}}
[9:02:01 PM] [Analyzer] [Info] <== {"event":"analysis.errors","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/example/android/app/src/debug/AndroidManifest.xml","errors":[]}}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"10","method":"analysis.setSubscriptions","params":{"subscriptions":{"CLOSING_LABELS":[],"FOLDING":[],"OCCURRENCES":[],"OUTLINE":[]}},"clientRequestTime":1587409337339}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"11","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/pubspec.yaml":{"type":"remove"}}},"clientRequestTime":1587409337339}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"12","method":"analysis.setPriorityFiles","params":{"files":[]},"clientRequestTime":1587409337339}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"13","method":"analysis.setSubscriptions","params":{"subscriptions":{"CLOSING_LABELS":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart"],"FOLDING":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart"],"OCCURRENCES":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart"],"OUTLINE":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart"]}},"clientRequestTime":1587409337379}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"14","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"content":"import 'dart:convert';\n\nimport 'package:meta/meta.dart';\nimport './authorization_request.dart';\n\n@immutable\nabstract class AuthorizationCredential {\n  const AuthorizationCredential();\n}\n\n/// An [AuthorizationCredential] which comes from a succesful Apple ID authorization.\n///\n/// Apple Docs: https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidcredential\nclass AuthorizationCredentialAppleID implements AuthorizationCredential {\n  const AuthorizationCredentialAppleID({\n    @required this.userIdentifier,\n    @required this.givenName,\n    @required this.familyName,\n    @required this.email,\n    @required this.identityToken,\n    @required this.authorizationCode,\n  }) : assert(authorizationCode != null);\n\n  /// An identifier associated with the authenticated user.\n  ///\n  /// This will be provided upon every sign-in.\n  /// This will stay the same between sign ins, until the user deauthorizes your App. TODO check if this is actually the case\n  ///\n  /// Can be `null`\n  final String userIdentifier;\n\n  /// The users given name, in case it was requested.\n  /// You will need to provide the [AppleIDAuthorizationScopes.fullName] scope to the [AppleIDAuthorizationRequest] for requesting this information.\n  ///\n  /// This information will only be provided on the first authorizations.\n  /// Upon further authorizations, you will only get the [userIdentifier],\n  /// meaning you will need to store this data securely on your servers.\n  /// For more information see: https://forums.developer.apple.com/thread/121496\n  ///\n  /// Can be `null`\n  final String givenName;\n\n  /// The users family name, in case it was requested.\n  /// You will need to provide the [AppleIDAuthorizationScopes.fullName] scope to the [AppleIDAuth…
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"15","method":"edit.getFixes","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","offset":0},"clientRequestTime":1587409337383}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"16","method":"edit.getAssists","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":0},"clientRequestTime":1587409337383}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"17","method":"edit.getAvailableRefactorings","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":0},"clientRequestTime":1587409337383}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"18","method":"analysis.setPriorityFiles","params":{"files":["/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart"]},"clientRequestTime":1587409337408}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"19","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"offset":0,"length":0,"replacement":"","id":""}],"type":"change"}}},"clientRequestTime":1587409337411}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"20","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"offset":0,"length":0,"replacement":"","id":""}],"type":"change"}}},"clientRequestTime":1587409337412}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"21","method":"edit.getFixes","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","offset":2622},"clientRequestTime":1587409337610}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"22","method":"edit.getAssists","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":2622},"clientRequestTime":1587409337610}
[9:02:17 PM] [Analyzer] [Info] ==> {"id":"23","method":"edit.getAvailableRefactorings","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":2622},"clientRequestTime":1587409337610}
[9:02:18 PM] [Analyzer] [Info] ==> {"id":"24","method":"edit.getFixes","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","offset":2774},"clientRequestTime":1587409338388}
[9:02:18 PM] [Analyzer] [Info] ==> {"id":"25","method":"edit.getAssists","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":2774},"clientRequestTime":1587409338388}
[9:02:18 PM] [Analyzer] [Info] ==> {"id":"26","method":"edit.getAvailableRefactorings","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":2774},"clientRequestTime":1587409338388}
[9:02:19 PM] [Analyzer] [Info] ==> {"id":"27","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"id":"","length":0,"offset":2774,"replacement":"\n  "}],"type":"change"}}},"clientRequestTime":1587409339409}
[9:02:19 PM] [Analyzer] [Info] ==> {"id":"28","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"id":"","length":0,"offset":2777,"replacement":"\n  "},{"id":"","length":2,"offset":2775,"replacement":""}],"type":"change"}}},"clientRequestTime":1587409339552}
[9:02:19 PM] [Analyzer] [Info] ==> {"id":"29","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"id":"","length":0,"offset":2778,"replacement":"\n  "},{"id":"","length":2,"offset":2776,"replacement":""}],"type":"change"}}},"clientRequestTime":1587409339709}
[9:02:19 PM] [Analyzer] [Info] ==> {"id":"30","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"id":"","length":0,"offset":2779,"replacement":"\n  "},{"id":"","length":2,"offset":2777,"replacement":""}],"type":"change"}}},"clientRequestTime":1587409339870}
[9:02:20 PM] [Analyzer] [Info] ==> {"id":"31","method":"edit.format","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","lineLength":80,"selectionLength":0,"selectionOffset":0},"clientRequestTime":1587409340423}
[9:02:27 PM] [Analyzer] [Info] ==> {"id":"32","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"offset":0,"length":0,"replacement":"","id":""}],"type":"change"}}},"clientRequestTime":1587409347226}
[9:02:27 PM] [Analyzer] [Info] ==> {"id":"33","method":"analysis.updateContent","params":{"files":{"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart":{"edits":[{"offset":0,"length":0,"replacement":"","id":""}],"type":"change"}}},"clientRequestTime":1587409347226}
[9:02:28 PM] [Analyzer] [Info] ==> {"id":"34","method":"analysis.getHover","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","offset":2769},"clientRequestTime":1587409348773}
[9:02:31 PM] [Analyzer] [Info] ==> {"id":"35","method":"analysis.getHover","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","offset":2720},"clientRequestTime":1587409351235}
[9:02:33 PM] [Analyzer] [Info] ==> {"id":"36","method":"edit.getFixes","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","offset":2720},"clientRequestTime":1587409353868}
[9:02:33 PM] [Analyzer] [Info] ==> {"id":"37","method":"edit.getAssists","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":2720},"clientRequestTime":1587409353868}
[9:02:33 PM] [Analyzer] [Info] ==> {"id":"38","method":"edit.getAvailableRefactorings","params":{"file":"/Users/henri.beck/Projects/AboutYou/dart_packages/packages/sign_in_with_apple/lib/src/authorization_credential.dart","length":0,"offset":2720},"clientRequestTime":1587409353868}
```

If you need more logs, just ask for them and I can provide them

FYI: We also have a project which uses Dart and the VSCode remote containers, and in that Remote Container the dart formatting and auto-completion works flawlessly.

**Comment by DanTup** [comment]
Can you capture an [analysis server instrumentation log](https://dartcode.org/docs/logging/#analyzer-instrumentation) with `dart.analyzerInstrumentationLogFile`?

How big is the `dart_packages` folder you've opened? Is it possible it contains symlinks that might result in infinite loops when traversed?

You can run `cd /Users/henri.beck/Projects/AboutYou/dart_packages && find . -type l -ls` in a terminal to print any symlinks.

**Comment by HenriBeck** [comment]
The `dart_packages` is rather small. A little more than 30 dart files. It's also not specific to that folder, I tried other/dart flutter and they had the same issue.

There are also only native symlinks in the `example` folders of the packages.

I did a `flutter clean` in the only package which actually has a proper flutter example, and this somehow fixed the issue. I have to say that the example app also contains a macOS flutter app and I did run it the day before this error occurred.

I will close this for now. I will give an update when I have a look again at macOS and Flutter and if I can reproduce my error again.
